protostar/io/output.py

```diff
@@ -0,0 +1,45 @@
+import json
+import sys
+from abc import ABC, abstractmethod
+
+from .log_color_provider import LogColorProvider
+
+
+class Message(ABC):
+    @abstractmethod
+    def format_human(self, fmt: LogColorProvider) -> str:
+        ...
+
+
+class StructuredMessage(Message, ABC):
+    @abstractmethod
+    def format_object(self) -> dict:
+        ...
+
+
+class Messenger(ABC):
+    @abstractmethod
+    def __call__(self, message: Message):
+        ...
+
+
+class HumanMessenger(Messenger):
```

Comment:
```suggestion
class ConsoleMessenger(Messenger):
```
Or `LoggerMessanger` maybe?

Reply:
> What's wrong with Human?

Sounds like we would have been sending messages from Mars (I think @Radinyn is a fan of such things) 😄 .

But seriously, we do not use human in other places of the project, we just call it `format` for example in `protostar/commands/test/test_result_formatter.py` so it is kinda inconsistent.  It should be marked Nitpick though.

Reply:
On the other hand we had no different types of `format` before. I think maybe human is alright but we have to be consistent with it.

Reply:
`protostar/commands/test/test_result_formatter.py` is up to be redone in scope of this project so it will naturally use `human`